source/content.js

```diff
@@ -9,9 +9,13 @@ import elementReady from './lib/element-ready';
 const errorMessage = 'npmhub: there was an error while ';
 
 async function fetchPackageFromNpm(name) {
-  const url = 'https://registry.npmjs.org/' + name.replace('/', '%2F');
-  const response = await fetch(url, {credentials: 'include'});
```

Comment:
Note: we used to send credentials to support private npm repos, but now I think `npm` won't receive that anymore via background.js

Note 2: `{credentials: 'include'}` is now the default of `fetch` anyway